runtime/standalone/Cargo.toml

```diff
@@ -146,6 +150,7 @@ std = [
   "pallet-identity/std",
   "pallet-clock/std",
   "circuit-runtime-types/std",
+  # "circuit-runtime-pallets/std",
```

Comment:
prolly just remove these comments until we need it since this brings in cumulus and everything that isn't required for standalone